types/styled-system/index.d.ts

```diff
@@ -843,6 +994,24 @@ export interface BoxShadowProps {
 
 export const boxShadow: styleFn;
 
+export interface TextShadowProps {
+    /**
+     * The `text-shadow` CSS property adds shadows to text. It accepts a comma-separated list of shadows to be applied
+     * to the text and any of its `decorations`. Each shadow is described by some combination of X and Y offsets from
+     * the element, blur radius, and color.
+     *
+     * [MDN reference](https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow)
+     */
+    textShadow?: ResponsiveValue<CSS.TextShadowProperty | number>;
+}
+
+export const textShadow: styleFn;
+
+export interface ShadowsProps
+    extends BoxShadowProps, TextShadowProps {}
+
+export const shadows: styleFn;
```

Comment:
Isn't this supposed to be `shadow` instead of shadows?
https://github.com/styled-system/styled-system/blob/master/packages/shadow/src/index.js#L14

Reply:
good catch! it should be

```suggestion
export const shadow: styleFn;
```

Reply:
the only place `shadows` plural should be is in themes